### PR TITLE
First draft of faqs

### PR DESCRIPTION
**Pull request by niallwingham**
Feel free to edit my copy or add other questions!

### CODE REVIEW

File: jobs/index.html [hunk 1]
```diff
@@ -26,5 +26,10 @@ <h3 id="benefits">Benefits</h3>
         </ul>
         <p>If something comes up for a team member that is not covered by one of these benefits, we will do our best to establish a new benefit to help them.</p>
         <p>Read more about our job philosophy <a href="/blog/2015/04/23/work-sustainably">here</a>.</p>
+
+        <h3 id="faq">FAQ</h3>
+        <p><strong>Is it really a 20-hour work week?</strong><br />Yes.  We only work 20 hours a week, whether it's billable or non-billable work.</p>
+        <p><strong>How much do you pay?</strong><br />We consider this a full-time job, so we offer a full-time salary and benefits.  We don't try to compete with Amazon or Facebook on salary, but we're probably at or above the median for software developers as a whole.</p>
```
Comment (Crisfole) [inline jobs/index.html]: I'd go with "We almost certainly pay above the US Median for software developers"

Comment (Crisfole) [inline jobs/index.html]: We're pretty much *at* median. Or I am. Tipping my hat here ;)

File: jobs/index.html [hunk 2]
```diff
@@ -26,5 +26,10 @@ <h3 id="benefits">Benefits</h3>
...
+        <p><strong>How much travel do you require?</strong><br />No travel is ever required.  Once a year we have a 3-4 day team retreat, but it's not obligatory.  Partners and family are also invited to attend, and everyone is paid for by the company.</p>
```
Comment (Crisfole) [inline jobs/index.html]: "You'll most likely be asked to come visit one of our clients within your first year. After that we have a 3-4 day team retreat yearly."

File: jobs/index.html (hunk 1 above)
Comment (niallwingham) [inline jobs/index.html]: Haha and then for Computer Programmer it's $79,840 (also BLS 2016 median).  I didn't know there was even a difference!

**Pull request merged**